app/controllers/application_controller.rb

```diff
@@ -1,10 +1,164 @@
 require './config/environment'
+require 'pry'
+
+# => {"username"=>"skittles123",
+#  "email"=>"skittles@aol.com",
+#  "password"=>"rainbows"}
+
+# => #<User:0x007faae80b9038
+#  id: 1,
+#  username: "skittles123",
+#  email: "skittles@aol.com",
+#  password: "rainbows">
 
 class ApplicationController < Sinatra::Base
```

Comment:
This controller is overloaded. Consider separating your concerns by splitting some of these actions into separate controllers (i.e. a users controller and a tweets controller).